mods/d2k/fluent/rules.ftl

```diff
@@ -375,11 +375,20 @@ actor-refinery =
     .description =
     Harvesters unload Spice here
     for processing.
+    Use Rotations hotkey for building rotation during placement
     .encyclopedia =
     The basis of all Spice production on Dune. Harvesters transport mined Spice to the Refinery where it is converted into credits. Refined Spice is automatically distributed to Silos and Refineries for storage. Each refinery can store Spice. A Spice Harvester is delivered by Carryall once a Refinery is built.
 
     Refineries are vulnerable to most weapons.
 
+
+actor-refinery_inverted =
+    .description =
+    Refinery with north/easth docking
+    Harvesters unload Spice here
+    for processing.
+    Use Rotations hotkey for building rotation during placement
```

Comment:
Add missing punctuation. Mind the toolip

<img width="374" alt="Screenshot 2025-05-30 at 15 38 29" src="https://github.com/user-attachments/assets/3d1346c5-d1c0-4f5c-a0cc-5e766c47b192" />

It looks quite ugly with such wrap

Reply:
See the image and how the text is wrapped. The tooltip size is dependent on text